### PR TITLE
data: Sunil8wale

### PR DESCRIPTION
**Pull request by Sunil8wale**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

You are doing a great job . I would like my LinkFree profile to be updated.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4770"></a>

### CODE REVIEW

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Comment by Sunil8wale** [comment]
Thank You So Much!!!

File: data/Sunil8wale.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Sunil Athawale",
+  "displayStatsPublic": true,
```
Comment (SaraJaoude) [inline data/Sunil8wale.json:3]: Following a recent improvement this is no longer needed
```suggestion
```

**Comment by eddiejaoude** [comment]
This is no longer required, but we can remove it
```
"displayStatsPublic": true,
```

**Pull request merged**